### ISSUE TITLE
Feature Request: Markdown links navigation

### ISSUE THREAD

**Issue by ysl2** [foambubble/foam#369]
Open links by hotkey: follow a wiki-link or a markdown link with “ctrl + enter”.

**Comment by ingalless** [comment]
I might be misremembering, but isn't this achievable by the "go to definition" shortcut? I think it's ctrl+f2 or ctrl+f12? (I use Vim for VSCode so for me it's `gd`)

**Comment by SanketDG** [comment]
Yup, the linking should be present already, @ysl970629 you could check your keybindings to see what keys "Go to Definition" is bound to.

The mouse hover preview is a great idea though.

**Comment by ysl970629** [comment]
> I might be misremembering, but isn't this achievable by the "go to definition" shortcut? I think it's ctrl+f2 or ctrl+f12? (I use Vim for VSCode so for me it's `gd`)

A wiki-link can be opened by this way, but a original markdown link can’t be.

**Comment by ingalless** [comment]
> > I might be misremembering, but isn't this achievable by the "go to definition" shortcut? I think it's ctrl+f2 or ctrl+f12? (I use Vim for VSCode so for me it's `gd`)
> 
> A wiki-link can be opened by this way, but a original markdown link can’t be. @ingalless 

Hmm, interesting! @riccardoferretti do you - recall if it's us or another extension providing this functionality?

**Comment by riccardoferretti** [comment]
We don't, and I don't believe the other extensions to do.
Once navigation is driven by the foam model, which will happen after #299, this should definitely be possible.

I also like the preview feature request. @ysl970629 can you please create a separate issue for it and rename this one to focus on the markdown links navigation?

**Comment by ysl970629** [comment]
And the original markdown links can’t be captured in “backlink” interface. 

In fact, as for my humble opinion, I prefer original links rather than wiki-links, because it could maintain portability (for example, to edit or view in a mobile phone, the wiki-links can’t be recognized well)

**Comment by riccardoferretti** [comment]
after #299, for Foam it won't matter whether you use regular links or wikilinks to connect documents, so you should be covered (well, once the backlinks panel is powered by the Foam model ;) )

**Comment by ysl970629** [comment]
OK, I have created a new issue about preview. Thank you :-) @riccardoferretti

**Comment by ysl970629** [comment]
Here is a solution for vim user: 
https://github.com/kortina/vscode-markdown-notes/blob/master/RECOMMENDED-SETTINGS.md#vscode-vim-settings-from-lukesmurray

@riccardoferretti @ingalless

**Comment by riccardoferretti** [comment]
This should be taken care of by #821
